<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stat_combo</class>
 <widget class="QDialog" name="stat_combo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(228, 230, 235);</string>
  </property>
  <widget class="QLabel" name="labelPourcentageNonValide">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 213, 224);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPourcentageValide">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 213, 224);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_stat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>491</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 213, 224);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
